eox_core/api/v1/views.py

```diff
@@ -137,6 +137,21 @@ def get(self, request, *args, **kwargs):
 
         return Response(response_data)
 
+    def patch(self, request, *args, **kwargs):
+        """
+        Partially update an user from edxapp. Not all the fields can be updated, just the ones thought as `safe`.
+        """
+        query = self.get_user_query(request)
```

Comment:
is this getting the information from the query params  ?

Reply:
Yes. This way we can use the method without adding new URLs and we can use the username or email through the query params. Do you think that is OK or is it better do define new paths like user/<username> or user/<email>?

Reply:
this is very odd, if you don't want to make new urls  you can get that information from the body

Reply:
I decided to add a new URL just for /user/<EMAIL>/. What do you think?